sockets.js

```diff
@@ -22,7 +22,7 @@ if (cluster.isMaster) {
 	var workers = exports.workers = {};
 
 	var spawnWorker = exports.spawnWorker = function () {
-		var worker = cluster.fork({PSPORT: Config.port, PSBINDADDR: Config.bindaddress || ''});
+		var worker = cluster.fork({PSPORT: Config.port, PSBINDADDR: Config.bindaddress || '', PSNOSSL: Config.ssl ? 0 : 1});
```

Comment:
Do we need `PSNOSSL`? If you do `node app.js 8001` you probably don't want SSL either. My current approach handles both situations.